.nvmrc

```diff
@@ -1 +1 @@
-14.17.1
+v16.11.0
```

Comment:
can we use generically "16"? we don't really care about minor/patch, as long as it's 16